app/routes/session.py

```diff
@@ -309,28 +301,18 @@ def get_signed_out() -> Response | str:
 
 def get_runner_claims(decrypted_token: Mapping[str, Any]) -> dict:
     try:
-        if version := decrypted_token.get("version"):
-            if version == AuthPayloadVersion.V2.value:
-                return validate_runner_claims_v2(decrypted_token)
+        return validate_runner_claims_v2(decrypted_token)
 
-            raise InvalidTokenException(f"Invalid runner claims version: {version}")
-
-        return validate_runner_claims(decrypted_token)
     except ValidationError as e:
-        raise InvalidTokenException("Invalid runner claims") from e
+        raise InvalidTokenException("Invalid runner claims version") from e
```

Comment:
Just thinking about this and had another question - is this exception message string now no longer related to the version? Would it instead just be a general issue with the claims now?

Reply:
Meant to remove this 😅